### PR TITLE
dcrjson: add fundrawtransaction command

### PR DESCRIPTION
**Pull request by vctt94**
related to https://github.com/decred/dcrwallet/pull/1195

### CODE REVIEW

**Comment by vctt94** [comment]
I changed to use just one parameter, now I believe this PR is ready for another review.

File: dcrjson/walletsvrcmds.go [hunk 1]
```diff
@@ -240,6 +240,33 @@ func NewEstimatePriorityCmd(numBlocks int64) *EstimatePriorityCmd {
 	}
 }
 
+// FundRawTransactionOptions represents the optional inputs to fund
+// a raw transaction.
+type FundRawTransactionOptions struct {
```
Comment (jrick) [inline dcrjson/walletsvrcmds.go]: This type must use `json:"..."` struct tags to provide the key name

File: dcrjson/walletsvrcmds.go [hunk 2]
```diff
@@ -240,6 +240,33 @@ func NewEstimatePriorityCmd(numBlocks int64) *EstimatePriorityCmd {
...
+	ChangeAccount         *string
```
Comment (jrick) [inline dcrjson/walletsvrcmds.go]: The change account is not part of the equivalent bitcoind RPC (instead, any address can be provided as an option).  Why is this different?

File: dcrjson/walletsvrcmds.go [hunk 3]
```diff
@@ -240,6 +240,33 @@ func NewEstimatePriorityCmd(numBlocks int64) *EstimatePriorityCmd {
...
+	LockUnspents          *bool `jsonrpcdefault:"false"`
```
Comment (jrick) [inline dcrjson/walletsvrcmds.go]: Let's not add this for now.  Locking outputs doesn't actually work in dcrwallet currently.

File: dcrjson/walletsvrcmds.go [hunk 4]
```diff
@@ -240,6 +240,33 @@ func NewEstimatePriorityCmd(numBlocks int64) *EstimatePriorityCmd {
...
+	FeeRate               *float64
+	RequiredConfirmations *int32
```
Comment (jrick) [inline dcrjson/walletsvrcmds.go]: bitcoind names this `conf_target`.

**Comment by vctt94** [comment]
fixed

**Pull request merged**